content/about/branching-and-merging.html

```diff
@@ -1,10 +1,9 @@
 ---
+order: 1
 section: "about"
 subsection: "branching-and-merging"
-layout: "about"
-title: "About - Git"
-sidebar: "book"
-branching_and_merging_class: "current"
```

Comment:
126a4569af3879da0d9286b43c3adfc98999ca6d:
This previous front matter seems kind of pointless now. Maybe this commit and 28c74c5923880494884b1827be9910ad58d577bf should be combined into one commit.

Reply:
I combined both commits into a single one.

Reply:
Here is the combined commit: fbba1286d